NuKeeper/Configuration/CommandLineArguments.cs

```diff
@@ -8,7 +9,8 @@ public class CommandLineArguments
         [CommandLine("mode", "m"), Required]
         public string Mode;
 
-        [CommandLine("github_token", "t"), Required, SensitiveInformation]
+        [Environment("NuKeeper_github_token"), Required, SensitiveInformation]
+        [OverriddenBy(ConfigurationSources.CommandLine, "t")]
```

Comment:
Both have their benefits - short when typing something quick, long for scripts, but it's not such a  big deal.